.github/workflows/deploy.yml

```diff
@@ -1,46 +1,34 @@
-# This file was created automatically with `myst init --gh-pages` 🪄 💚
-
 name: MyST GitHub Pages Deploy
 on:
+  pull_request:
+    branches: [main]
   push:
     # Runs on pushes targeting the default branch
     branches: [main]
-env:
-  # `BASE_URL` determines the website is served from, including CSS & JS assets
-  # You may need to change this to `BASE_URL: ''`
-  BASE_URL: /${{ github.event.repository.name }}
-
-# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
-permissions:
-  contents: read
-  pages: write
-  id-token: write
-# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
-# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
-concurrency:
-  group: "pages"
-  cancel-in-progress: false
 jobs:
-  deploy:
-    environment:
-      name: github-pages
-      url: ${{ steps.deployment.outputs.page_url }}
+  build:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
       - name: Setup Pages
-        uses: actions/configure-pages@v3
-      - uses: actions/setup-node@v4
+        uses: actions/configure-pages@v5
         with:
-          node-version: 18.x
-      - name: Install MyST Markdown
-        run: npm install -g mystmd
-      - name: Build HTML Assets
-        run: myst build --html
-      - name: Upload artifact
-        uses: actions/upload-pages-artifact@v1
+          enablement: "true"
+      - uses: prefix-dev/setup-pixi@v0.7.0
         with:
-          path: "./_build/html"
-      - name: Deploy to GitHub Pages
-        id: deployment
-        uses: actions/deploy-pages@v2
+          path: ./_build/html
+
```

Comment:
There's currently no build step defined. Can you define a `pixi` task called `doc`?
https://pixi.sh/latest/features/advanced_tasks/